src/functionalTest/java/uk/gov/hmcts/idam/userprofilebridge/steps/UserSteps.java

```diff
@@ -45,4 +63,26 @@ private User createUser(ActivatedUserRequest activatedUserRequest) {
             .and().extract().as(User.class);
     }
 
+    private User updateUser(ActivatedUserRequest activatedUserRequest) {
+        String token = getTestingServiceClientToken();
+        return SerenityRest.given().baseUri(EnvConfig.TESTING_SUPPORT_API_URL)
+            .header("authorization", "Bearer " + token)
+            .contentType(ContentType.JSON)
+            .body(activatedUserRequest)
+            .put("/test/idam/users/" + activatedUserRequest.getUser().getId())
+            .then().assertThat().statusCode(HttpStatus.OK.value())
+            .and().extract().as(User.class);
+    }
+
+    private User createCftUser(ActivatedUserRequest activatedUserRequest) {
+        String token = getTestingServiceClientToken();
+        return SerenityRest.given().baseUri(EnvConfig.TESTING_SUPPORT_API_URL)
+            .header("authorization", "Bearer " + token)
+            .contentType(ContentType.JSON)
+            .body(activatedUserRequest)
+            .put("/test/cft/users/" + activatedUserRequest.getUser().getId())
```

Comment:
I think I've missed idam-testing-support-api's PR for that new endpoint, apologies. 
It's a bit tricky relying on the difference between "IDAM User" and "CFT User" - lots of people will use them to mean the same thing.
What's the difference in that context?

Reply:
The "cft" user exists in both idam and ref data